lib/ReactTags.js

```diff
@@ -203,6 +204,25 @@ ReactTags.propTypes = {
   suggestions: PropTypes.array,
   autofocus: PropTypes.bool,
   autoresize: PropTypes.bool,
+  delimiters: function (props, propName, componentName) {
```

Comment:
Nice, it's too bad it needa to do the array validation, I was going to suggest :

```
  // You can also supply a custom validator to `arrayOf` and `objectOf`.
  // It should return an Error object if the validation fails. The validator
  // will be called for each key in the array or object. The first two
  // arguments of the validator are the array or object itself, and the
  // current item's key.
  customArrayProp: PropTypes.arrayOf(function(propValue, key, componentName, location, propFullName) {
    if (!/matchme/.test(propValue[key])) {
      return new Error(
        'Invalid prop `' + propFullName + '` supplied to' +
        ' `' + componentName + '`. Validation failed.'
      );
    }
  })
```

(from https://facebook.github.io/react/docs/typechecking-with-proptypes.html), but if the array is empty the custom validation isn't triggered.